apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/Version2CustomTypeAdapter.kt

```diff
@@ -51,3 +53,19 @@ sealed class CustomTypeValue<T>(@JvmField val value: T) {
     }
   }
 }
+
+/**
+ * A replica of Apollo Android v2's ScalarTypeAdapters, to ease migration from v2 to v3.
```

Comment:
Nitpick: not sure 'ease migration' is the right word for this one since it will throw - OTOH I don't have a better wording either :)